test/home_widget_test.dart

```diff
@@ -47,6 +47,10 @@ void main() {
           return Future.value(launchUri);
         case 'registerBackgroundCallback':
           return true;
+        case 'requestPinWidget':
+          return true;
```

Comment:
Should maybe return `null` in order to mimic better what the actual result is doing.